src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,98 @@
+# archive_unstable_storageDiff
+
+**Parameters**:
+
+- `hash`: String containing a hexadecimal-encoded hash of the header of the block whose storage difference will be retrieved.
+
+- `previousHash` (optional): String containing a hexadecimal-encoded hash of the header of the block from which the storage difference will be calculated. The `previousHash` must be an ancestor of the provided `hash`.  When this parameter is provided, the storage difference is calculated between the `hash` block and the `previousHash` block. If this parameter is not provided, the storage difference is calculated between the `hash` block and the parent of the `hash` block.
+
+- `items` (optional): Array of objects. The structure of these objects is found below.
+
+    ```json
+    "items": [
+        {
+            "prefixes": [
+                {
+                    "key": "0x...",
+                    "type": "value" | "hash" | "none",
+                },
+            ],
+
+            "trieType": "mainTrie" | "childTrie",
+            "childTrieKey": "0x...",
+        },
```

Comment:
It feels a little off to me that we have two arrays nested inside eachother in order to presumably support child tree access. I wonder whether we could simplify itms to be:

```json
"items": [
    {
        "key": "0x...",
        "type": "value" | "hash" | "none",
        "childTrieKey": "0x...",
    }
]
```

If "childTrieKey" is provided then we try accessing said child trie, else we assume we're trying to access the main trie (which I guess is the overwhelming default for most, but not 100% sure!).

Reply:
Yep that makes sense! While at it, I've made `key` and `childTrie` optional